themes/default/content/docs/intro/concepts/resources/options/providers.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Providers"
+meta_desc: An in depth look at Pulumi resources and their usage.
+menu:
+  intro:
+    parent: options
+    weight: 10
```

Comment:
This should probably be changed to `11` (`10` is already in use by the provider page), and then `replaceonchanges.md`, `transformations.md`, and `version.md` changed to `12`, `13`, and `14`, respectively. Otherwise, this page won't appear in the left-hand nav.